### PR TITLE
Stop TravisCI to upload deb to Bintray- on-dhcp-proxy

### PR DESCRIPTION
**Pull request by panpan0000**
because it's taken over by Jenkins with better version control and test

all background/detail/test , please refer to https://github.com/RackHD/on-tftp/pull/56 for detail.

### CODE REVIEW

**Comment by panpan0000** [comment]
test this please

**Comment by panpan0000** [comment]
test this please

**Comment by panpan0000** [comment]
test this please

**Comment by panpan0000** [comment]
test this please

**Comment by panpan0000** [comment]
test this please

**Comment by panpan0000** [comment]
test this please

File: .travis.yml [hunk 1]
```diff
@@ -18,18 +18,9 @@ addons:
 
 after_success:
  - ./extra/make-testcoveralls.sh
-
-before_deploy:
+ # below is the placeholder to leverage TravisCI to verify building capability is good. will be replaced by npm package build.
  - ./extra/make-deb.sh
 
-deploy:
```
Comment (anhou) [inline .travis.yml]: .bintray.json is still there

File: .travis.yml [hunk 2]
```diff
@@ -18,18 +18,9 @@ addons:
 
 after_success:
  - ./extra/make-testcoveralls.sh
-
-before_deploy:
+ # below is the placeholder to leverage TravisCI to verify building capability is good. will be replaced by npm package build.
```
Comment (anhou) [inline .travis.yml]: personally, I think if it's not for coverage or generate any packages for RackHD, don't need to verify the building process here, just do it in the build places.

**Comment by panpan0000** [comment]
close this PR, because the PR is created from wrong repo(RackHD-mirror, instead of my personal repo...)

**Pull request closed without merging**